builder/azure/common/template/template_builder.go

```diff
@@ -450,6 +450,22 @@ func (s *TemplateBuilder) SetBootDiagnostics(diagSTG string) error {
 	return nil
 }
 
+func (s *TemplateBuilder) SetSecurityProfile(secureBootEnabled bool, vtpmEnabled bool) error {
```

Comment:
is the destination image also being marked correctly as trusted launch?

Reply:
A managed image has no specific setting. In case of the destination being a shared image gallery it won't set these properties that would need additional settings to be applied when publishing the image to to the gallery. But you can take an already created trusted launch image from a shared image gallery as source image

Reply:
hmm it seems now that Trusted Launch VMs cannot be saved as managed images anymore, only direct publish to SIG is supported

Reply:
Yes, there is a PR in progress. I think you will have to wait until that is done.

Reply:
Hi There, is there any ideas as to when this enhancement will be generally available and useable in the wild?

Reply:
hi there, +1 on this enhancement. excitedly waiting in anticipation